task.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      
+      <meta charset="UTF-8">
+      <meta name="viewport"  ,initial-scale=1.0 >
+      <link rel="stylesheet" href="task.css">
+    <title>TASK </title>
+  </head>
+  <body>
+    <div class="container">
+        <div class="first">
+    <h1>  Remit  </h1><br><br><br>
+    <h2> Welcome </h2>
+    <p>  Sign in by entering the information below  </p><br><br>
+      <div class="wrap">  
+        <div class="in">
+             
+    <label  > Email Address</label>
+    <input type="email" id="email" placeholder="username@gmail.com" required ><br><br>
+    
+    <label  > Password</label>
+    <input type="password" id="pass" placeholder="Enter the password"  required><br><br>
+</div>
+</div> 
+ <div class="lab">
+<br><br><br><br><br><br>
+    <input type="checkbox" id="rem" name="rem" value="rem-me">

```

Comment:
Remember me should be grouped in a `div` and forget password should be grouped into another group
```html
<div class="remember-me">
    <label for="rememberMe">Remember Me</label>
    <input type="checkbox" id="rememberMe" name="rememberMe" value="rememberMe">
</div>
<div class="forgot-password">
    <label for="forgotPassword">Forgotten password</label>
    <input type="checkbox" id="forgotPassword" name="forgotPassword" value="forgotPass">
</div>
```